app/lib/database/models/user/user_log.dart

```diff
@@ -0,0 +1,28 @@
+import 'package:hive/hive.dart';
+import 'package:so_tired/utils.dart';
+import 'package:so_tired/database/models/user/user_access_method.dart';
+import 'package:so_tired/database/models/user/user_game_execution.dart';
+
+part 'user_log.g.dart';
+
+/// This class defines the [UserLog] model by extending a [HiveObject].
+/// It holds an [uuid], a [accessMethod] and [Map] defining which game the user
+/// has executed.
+/// The UUID can be generated using the [Utils] class.
+@HiveType(typeId: 1)
+class UserLog extends HiveObject {
+  @HiveField(0)
+  String? uuid;
+
+  @HiveField(1)
+  UserAccessMethod? accessMethod;
+
+  @HiveField(2)
+  Map<UserGameExecution, Map<String, dynamic>>? gameExecution;
+
```

Comment:
Please add a Timestamp field to track the execution time. This is a valuable information for the study especially to find correlations between the execution of app modules and the notification time.